app/models/apitag.rb

```diff
@@ -0,0 +1,4 @@
+class Apitag < ActiveRecord::Base
+  belongs_to :api
+  belongs_to :tag
+end
```

Comment:
This class should be called `ApiTag`, the database table should be called `api_tags`, and your model relationships should be updated.